vite.config.js

```diff
@@ -14,6 +14,11 @@ export default defineConfig(({mode}) => {
             outDir: "../../../target",
             minify: true,
             sourcemap: true,
+            rollupOptions: {
+                output: {
+                    assetFileNames: (asset) => asset.name.includes("config.json") ? "assets/[name][extname]": "assets/[name].[hash][extname]",
```

Comment:
we use this option elsewhere as well to not hash the theme manifests, right? do we have to take that into account here or not?

Reply:
I haven't created that PR yet. Those changes will be in the theme chooser PR.